mne_bids/mne_bids.py

```diff
@@ -68,6 +68,17 @@
                     'Biosemi': ['STI 014'],
                     'Neuroscan': ['STI 014']}
 
+COORDINATE_FRAMES = {FIFF.FIFFV_COORD_UNKNOWN: 'Unknown',
```

Comment:
let's make this a private variable?

Reply:
you mean defined locally within `_sidecar_json`?

Reply:
I mean it should be preceded by an underscore. So that it's designated as a private variable not accessible to the user

Reply:
because I just realized it now they should all be private :-) But I don't think we can change things now because we've already made a release ...